### ISSUE TITLE
Incompatible with Babel 7

### ISSUE THREAD

**Issue by EthanEFung** [javascriptiscoolpl/npm-simple-react-pdf#10]
Hi, is this module still being maintained?
I've been using simple pdf for our project, and have migrated dependencies from babel 6 -> 7. Now our React project can't compile. Here are our specs:
```
"devDependencies": {
    "@babel/cli": "7.4.4",
    "@babel/core": "7.4.4",
    "@babel/preset-react": "7.0.0",
    "babel-loader": "8.0.6",
},
"dependencies: {
    "pdfjs-dist": "2.0.943",
    "react": "15.3.1",
    "react-dom": "15.3.1",
}
```
Here is our error:
<img width="856" alt="Screen Shot 2019-05-16 at 9 34 41 PM" src="https://user-images.githubusercontent.com/24540371/57903490-2ca96c00-7823-11e9-94fc-fc0f2f5e2228.png">

**Comment by javascriptiscoolpl** [comment]
I will try to look more closely to this issue ASAP, maybe this weekend. Stay tuned, let you know if this problem is related to the module build itself or not.

**Comment by javascriptiscoolpl** [comment]
Hello again. I was able to run and rebuild demo without any errors. 
There are two warnings related to deprecation: 
```
npm WARN deprecated core-js@1.2.7: core-js@<2.6.5 is no longer maintained.
npm WARN deprecated babel-preset-es2015@6.24.1
```
but this not stop working or rebuilding. I will try to upgrade this package with latest versions but do not want to force people to use latest versions. 

Your error looks like You use wrong path. There is no: `simple-react-pdf/node_modules`. Check the demo code and path there and let me know if this is still issue.

**Comment by EthanEFung** [comment]
Right, that makes sense since I'm sure there are a few legacy projects that are using your package. Since our company is migrating to babel 7 however, our project is no longer compatible with this package. Babel made non-trivial framework changes moving from 6 to 7. Removing the yearly and stage presets as well as switching from `babelrc` to `babel.config.js` files to set a projects configuration.

Maybe because of the`babelrc` and `babel.config.js` switch our project no longer ignores simple-react-pdf's node_modules.

If we need to we can try and replace our simple-pdf package with another react-pdf module, but we really like simple-react-pdf. It's easy to use.

**Comment by javascriptiscoolpl** [comment]
Hi again, **I am pretty sure You don't have to rebuild `simple-react-pdf` package to use it in Your project.** 

The Demo is the example how to use the package as it is without rebuilding it. There is no `simple-react-pdf/node_modules` directory until You trying to rebuild the package itself, I guess there is no such need to build Your project with new Babel version and use this package as it is. 

If You feel disappointed, feel free to make fork and add provide needed changes for Your special project. However I will try to check it again and rebuild the package with new Babel version and then play a little bit more with the Demo to make sure everything is fine. Stay tuned.

**Comment by javascriptiscoolpl** [comment]
OK. Here is the solution, I guess. For Your project You just need: 
```
"dependencies": {
    "react": "^15.0.0",
    "react-dom": "^15.0.0",
    "simple-react-pdf": "^1.0.3"
  },
  "devDependencies": {
    "@babel/cli": "7.4.4",
    "@babel/core": "7.4.4",
    "@babel/preset-react": "7.0.0",
    "babel-loader": "8.0.6",

```

**Additional info:** 
- Don't add `pdfjs-dist` in `package.json` dependencies.
- You can build Your project with **any** Babel version.
- Using any module in npm You don't have to rebuild it, also You don't need to rebuild `simple-react-pdf`, this is how modularity idea looks like, You build something and You just use it later on as a module.
- You can remove everything from `node_modules/simple-react-pdf` directory except `demo` folder and start Your project just from the `demo` folder.
- For serious projects You should rather use [simple-react-pdf-service](https://github.com/javascriptiscoolpl/npm-simple-react-pdf-service)
- You are always welcome to make fork.

**Comment by EthanEFung** [comment]
Thanks @javascriptiscoolpl , I appreciate this. I'll try building it as you've directed.

**Comment by javascriptiscoolpl** [comment]
I can tell You that I was trying to rebuild the module and demo with all new Babel version and I was able to build the module but I found issue with browserify and babelify (react import or something). I guess the CLI tools are not fully ready for new versions yet. 

For sure I will remove all deprecation issues and other warnings, to look more beauty, however I don't want to add webpack dependency to not oversize the package. 

But as I mentioned before, the modularity idea is to not build things again and again, modules are small things to use it. Good luck and feel free to open issues or make forks.
